fsliolib/CMakeLists.txt

```diff
@@ -10,23 +11,15 @@ if(BUILD_SHARED_LIBS)
   set_target_properties(fslio PROPERTIES ${NIFTI_LIBRARY_PROPERTIES})
 endif()
 
-if(NOT NIFTI_INSTALL_NO_LIBRARIES)
```

Comment:
why is this removed?

Reply:
I was trying to drop the number of variables to define. The COMPONENTS option in the install command allows a user to install their choice of executables,libraries, and header files. Happy to change this back if this wreaks havoc elsewhere though